style.css

```diff
@@ -11,3 +11,46 @@ body {
 .display-off {
 	display: none;
 }
+
+/*Section -1*/
+.section-1 {
+	display: flex;
+	height: 50px;
+	justify-content: center;
+}
+.section-1-main_box {
+	display: flex;
+	justify-content: space-evenly;
+	width: 50%;
+}
+
+.reset-box {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+}
+.display-box,
+.reset-box {
+	display: flex;
+	align-items: center;
+}
+#reset {
+	background-color: #dc3545;
+	color: #fff;
+	padding: 0.375rem 0.75rem;
+	font-size: 1rem;
+	border-color: #dc3545;
+	border: 1px solid transparent;
+	border-radius: 0.25rem;
+	font-weight: 600;
+}
+#display {
+	color: #fff;
+	background-color: #007bff;
+	border-color: #007bff;
+	padding: 0.375rem 0.75rem;
+	font-size: 1rem;
```

Comment:
I would personally make the buttons a bit bigger , what do you think?

Reply:
@aleks2407  after Oguz part maybe I can rearrange.

Reply:
I think the greatness looks good, if it is too big it can look bad and @mametur  you can do `border-style: none;` features of display button

Reply:
@oguzkarademir  yeah it is kind of none `solid transparent` :)